spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,21 @@
+
+require 'pg'
+
+feature 'Viewing Peeps' do
+  scenario 'visiting the index page' do
+    visit ('/')
+    expect(page).to have_content "Chitter"
+  end
+
+  scenario 'A user can see bookmarks' do
+    Peep.create(content: "Peep 1")
+    Peep.create(content: "Peep 2")
+    Peep.create(content: "Peep 3")
```

Comment:
feature tests are more about simulating stuff the user can do. so rather than doing `Peep.create` you should use Capybara to simulate entering a peep via the front end, e.g. something like:
```ruby
```

Reply:
On second thoughts I'm not totally sure about this... maybe it's fine how you've got it...

Reply:
according to [the rubric](https://github.com/makersacademy/chitter-challenge/blob/master/docs/review.md#ensure-correct-location-of-featureacceptance-and-unit-tests), in a feature test:

>  It's acceptable to manipulate the database directly to set things up, but note their are risks associated here, i.e. that you will get your database in a state that it couldn't get into via the web interface, then when the tests pass it may not reflect precisely the user experience of using the site.